src/components/scene/inspector.js

```diff
@@ -0,0 +1,52 @@
+/* global AFRAME */
+var registerComponent = require('../../core/component').registerComponent;
+var pkg = require('../../../package');
+
+var INSPECTOR_URL = pkg.homepage + 'releases/' + pkg.version + '/aframe-inspector.min.js';
+
+module.exports.Component = registerComponent('inspector', {
+  schema: {
+    url: {default: INSPECTOR_URL}
+  },
+
+  init: function () {
+    this.onKeydown = this.onKeydown.bind(this);
+  },
+
+  play: function () {
+    window.addEventListener('keydown', this.onKeydown);
+  },
+
+  pause: function () {
+    window.removeEventListener('keydown', this.onKeydown);
+  },
+
+  onKeydown: function (evt) {
+    // Alt + Ctrl + e
+    var shortcutPressed = evt.keyCode === 69 && evt.ctrlKey && evt.altKey;
```

Comment:
there might be some form field on the page. so this should be checking if we should be listening for the keypress. you can either use [our `utils.shouldCaptureKeyEvent` method](https://github.com/aframevr/aframe/blob/master/src/utils/index.js#L127-L137).

what was that case when you said `document.activeElement` was not `<body>` in some browser in some case? I can't reproduce that issue you were talking about.